src/caffe/proto/caffe.proto

```diff
@@ -347,6 +347,9 @@ message LayerParameter {
   // Parameters shared by loss layers.
   optional LossParameter loss_param = 101;
 
+  // Specify name of source class
+  optional string source_class = 143;
```

Comment:
Agree. Also, this parameter (along with `source`) needs some more explanation, as right now they seem unused in Caffe. Why are we adding these?